test/common_nn.py

```diff
@@ -46,12 +46,14 @@ def get_weight(m):
     dict(
         module_name='Linear',
         constructor_args=(10, 8),
+        cpp_constructor_args='(10, 8)',
```

Comment:
You don't need to do this, but we should really turn this into a namedtuple at some point